build.yaml

```diff
@@ -427,9 +427,9 @@ common_cli:
 # flocker.node.functional is hanging, so we don't run it
 job_type:
   run_trial:
-  # http://build.clusterhq.com/builders/flocker-centos-7
+    # http://build.clusterhq.com/builders/flocker-centos-7
```

Comment:
nope, that's just me indenting the comment to match the line it refers to below that comment.
Not really vi related, the top line 'the modeline' simply says 2 spaces no tabs.
Keep in mind this is YAML not python